src/main/java/org/folio/consortia/domain/entity/SharingPolicyEntity.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.consortia.domain.entity;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.folio.consortia.domain.entity.base.AuditableEntity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity
+@Table(name = "sharing_policy")
+public class SharingPolicyEntity extends AuditableEntity {
+  @Id
+  private UUID id;
+  private UUID policyId;
+  private String tenantId;
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Lombok annotation @EqualsAndHashCode will generate the same code. Any reason for not using it?

Reply:
Not big deal, but there is one small reason for this. @EqualsAndHashCode has two approach @EqualsAndHashCode(callSuper=false), @EqualsAndHashCode(callSuper=true). Most entitties extended from AuditableEntity class, and usually ideas warn @EqualsAndHashCode annotation. There might be case somone wrongly configure @EqualsAndHashCode(callSuper=true) cases which compare with superClass as well that is incorrect. Manual implementation as a standart approach chosen for consortia.

Reply:
Overall, using annotation also not bad approach, but need to implement for other entities as well.